### ISSUE TITLE
HTML5 video livestreams not playing in linuxbrowser

### ISSUE THREAD

**Issue by JonnyTech** [bazukas/obs-linuxbrowser#73]
How can I play HTML5 video such as youtube streams?

**Comment by NexAdn** [comment]
HTML5 Video in general is working. However, MPEG is a proprietary codec which is per default not supported by CEF (but often used for livestreams). You'd have to compile CEF with proprietary codecs enabled to be able to view MPEG video.

YouTube videos are unaffected by this as YouTube transcodes videos to other formats like WebM when processing newly uploaded videos, but in livestreams there isn't enough time for that to happen.

TL;DR Either use videos only (or Non-MPEG sources), or compile CEF with proprietary codecs enabled.

**Comment by JonnyTech** [comment]
Thank you for the detailed reply
